ubuntu/debian/libignition-rendering-ogre1-dev.install

```diff
@@ -3,4 +3,5 @@ usr/include/ignition/rendering[0-99]/ignition/rendering/ogre.hh
 usr/lib/*/libignition-rendering[0-99]-ogre.so
 usr/lib/*/pkgconfig/ignition-rendering[0-99]-ogre.pc
 usr/lib/*/cmake/ignition-rendering[0-99]-ogre/*
+usr/lib/*/ign-rendering-[0-99]/engine-plugins/libignition-rendering*-ogre*.so
```

Comment:
This will install `ogre2` as well, right?

Reply:
I dont see a ogre-dev.so  or files with other suffix strings so I think removing the asterisk works. 65d975fe6dc090d495126884285e639de449fbb1